tests/ReproduceTest.php

```diff
@@ -0,0 +1,195 @@
+<?php
+/*
+ *
+ * Copyright 2018 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+session_start();
+header('Content-type: text/plain');
+
+require_once(__DIR__ . '/vendor/autoload.php');
+require_once(__DIR__ . '/../vendor/autoload.php');
```

Comment:
Two things here:

1. We do not need two composer installations. Use `require-dev` for any test dependencies, and this will ensure the dependencies are not installed in production or by users installing this package.
2. Include a `bootstrap` file so that you do not need to require the autoload file at the top of every test. See [here](https://github.com/GoogleCloudPlatform/google-cloud-php/blob/master/phpunit.xml.dist#L3) for an example. You will want to create a `phpunit.xml.dist` file for this, which is standard. Also, the bootstrap file can BE the `vendor/autoload.php` file, for simplicity.

Reply:
I am going to add `bootstrap` by another PR, which will clean all test files under the `test/` directory. Thank you for this instruction. It is very useful!